integration-tests-ex/cases/cluster/Common/prepopulated-data.env

```diff
@@ -0,0 +1,24 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+AWS_REGION=us-east-1
+
+# Setting s3 credentials and region to use pre-populated data for testing.
+druid_s3_accessKey=AKIAT2GGLKKJQCMG64V4
+druid_s3_secretKey=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
```

Comment:
Do we want to check these keys into GitHub? If so, then we should have a comment that explains that, say, they point to a read-only public bucket, or whatever the story is. This will help folks who do a security scan and stumble across these keys.

Reply:
These are already in present in the old framework. See https://github.com/apache/druid/blob/master/integration-tests/docker/environment-configs/test-groups/prepopulated-data.
So I assumed there will be no issues.

Reply:
You are probably right. Perhaps include a comment saying that these are for a public repo and OK to check in. It will save the security folks from having to ask around to figure out if we have an issue. Maybe just:

```text
# These keys are for a public, read-only bucket. The keys are intended for public use.
```